src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -31,40 +38,42 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
         className={classNames({ "menu-open": isOpen, active: item.children?.find((s) => s.active) || item.active })}
```

Comment:
What do you think about moving this to a variable insted of having duplicated code ? `item.children?.find((s) => s.active) || item.active })}`

Reply:
@manni497 Moved into a `isActive` variable